README.md

```diff
@@ -4519,6 +4519,17 @@ $ curl https://mainnet.aeternity.io/mdw/v2/accounts/ak_2nVdbZTBVTbAet8j2hQhLmfNm
         "micro_index": 0
       },
       "type": "InternalContractCallEvent"
+    },
+    {
+      "height": 653289,
+      "payload": {
+        "from": "ak_11111111111111111111111111111111273Yts",
+        "index": 0,
```

Comment:
`log_index` would make it clearer and good idea to omit the `call_txi`, would do the same for `InternalContractCallEvent`

Reply:
Done